packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -260,6 +260,11 @@
                             <oui-clipboard
                                 model="$ctrl.endpoint.uri"
                             ></oui-clipboard>
+                            <span
+                                style="word-break: break-word;"
```

Comment:
**nitpick:** Inline stylings must not be used
```suggestion
                                class="word-break"
```

Reply:
Definitely better, thanks ! https://github.com/ovh/manager/pull/7460/commits/d4d4bed42d64929ae00b322f1c38f3b297c4bdb0